### ISSUE TITLE
Steps for training a model on custom data

### ISSUE THREAD

**Issue by DhavalThkkar** [Layout-Parser/layout-parser#5]
There are no steps to understand how to train a model on a custom dataset. I'm really looking to use this in my current workflow and kudos for developing this!

**Comment by lolipopshock** [comment]
Thanks @DhavalThkkar ! We are working on this topic and you could find some ready-to-use scripts in this repo https://github.com/Layout-Parser/layout-model-training . Please stay tuned for more updates.

**Comment by abdksyed** [comment]
Any updates on fine tuning available models on custom dataset??

**Comment by kunalkonoha** [comment]
Hi Team,
Is there any plan to release this ?

**Comment by deshwalmahesh** [comment]
You can [See this kaggle tutorial](https://www.kaggle.com/code/ammarnassanalhajali/layout-parser-model-training?scriptVersionId=77893632&cellId=13) but the thing is that there is not enough information given on why some specific things have been done.

**Comment by SimNaik** [comment]
yeah that's right I have a custom dataset but I need to fine tune the model and change its hyper parameter, but there is not enough information on this
